.gitignore

```diff
@@ -1,5 +1,69 @@
-SquareFlowLayout/Example/SquareFlowLayout/SquareFlowLayoutExample.xcodeproj/xcuserdata/
-SquareFlowLayout/Example/SquareFlowLayout/SquareFlowLayoutExample.xcworkspace/xcshareddata/
-SquareFlowLayout/Example/SquareFlowLayout/SquareFlowLayoutExample.xcworkspace/xcuserdata/
-SquareFlowLayout/Example/SquareFlowLayout/SquareFlowLayoutExample.xcodeproj/project.xcworkspace/
-SquareFlowLayout/Example/SquareFlowLayout/Pods/
\ No newline at end of file
+# Xcode
```

Comment:
I prefer to have well ordered `gitignore` with only necessary rules, it's make it much easy to manage it